src/action/emailVerification/emailVerificationAction.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-constant-condition */
+import makeRequest from '../../utils/axiosSetup';
+
+export const UPDATE_STATUS_CODE = 'UPDATE_STATUS_CODE';
+
+export const sendConfirmationMailApiCall = email => {
+  return async dispatch => {
+    try {
+      await makeRequest(`/auth/resend-verification-link`, {
+        method: 'POST',
+        body: { email }
+      });
+      dispatch({ type: 'ON_EMAIL_SUBMIT', payload: { error: '' } });
```

Comment:
Hello Tejiri, nice work on this task.

However, it seems you are expecting a payload of the error object here and in the catch block. I thought this should be like the success data you are expecting.

Reply:
I removed the error payload but was not expecting any data as the user will be redirected even if an error occurs.